pom.xml

```diff
@@ -250,6 +251,7 @@
                               <exclude>**/src/site/resources/**/*.txt</exclude>
                               <exclude>**/src/site/resources/**/*.json</exclude>
                               <exclude>**/.gitignore</exclude>
+                              <exclude>**/*.md</exclude>
```

Comment:
I  think excludes that include ** and make sense across the project would go in streams master, and the exclusion of specific files for specific reasons would be in the modules